packages/dds/merge-tree/src/attributionCollection.ts

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+/* eslint-disable unicorn/no-null */
```

Comment:
I would recommend disabling this rule on a line-by-line bases as needed, instead of disabling it globally. That way new usages are not introduced where they shouldn't be.